### PR TITLE
New component: TimerPulser

### PR DESCRIPTION
**Pull request by TaraldRotsaert**
### Description
- Add a TimerPulser animation component

#### Screenshot after this PR
Screen Shot 2019-03-25 at 10 19 29

### Breaking changes
- None

### CODE REVIEW

File: src/components/timerPulser/theme.css [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+  position: relative;
+  z-index: 3;
+  &::after {
+    content: '';
+    width: 100%;
+    height: 100%;
+    background: white;
+    display: block;
+    position: absolute;
+    z-index: -2;
```
Comment (kevinwaelkens) [inline src/components/timerPulser/theme.css]: Q: Won't this z-index act up weirdly with other regularly z-indexed items now?

**Comment by TaraldRotsaert** [comment]
Stephen taught it would be a good addition to our library since we are going to reuse the animation in the future.

**Comment by duivvv** [comment]
Wouldn't it make more sense to think about re-using the animation only then? (maybe a HOC or a render-prop component)

Now we're adding a very specific stop button with animation in UI?

**Comment by kevinwaelkens** [comment]
I'm also a little surprised by this addition tbf 😬

**Comment by ArnaudWeyts** [comment]
Agree with @lowiebenoot and @duivvv here, I don't think this should be a ui-component, I can understand that you'd want to re-use the animation but then it belongs in [ui-animation](https://github.com/teamleadercrm/ui-animations).

**Comment by mikeverf** [comment]
So what's happening with this PR, @TaraldRotsaert? Are you adding it to `ui-animations` and closing here? Or do you really need this entire component to be in ui?

**Comment by TaraldRotsaert** [comment]
Somewhere today or monday Stephen and i are going to look into it and make a decision.

**Comment by stephen-tl** [comment]
Discussed this with @TaraldRotsaert and we should add this to the styleguide. It is a dedicated part of our time tracking patterns and will be needed for other components that involve time tracking.

**Comment by TaraldRotsaert** [comment]
We are not going to integrate the animation in our `ui-animations` library but i am going to expand this component so it has more options and can be displayed in several ways.

**Comment by kevinwaelkens** [comment]
🚨 	Conflicts need to be fixed first

**Pull request merged**